content/en/docs/ambient/getting-started/cleanup/index.md

```diff
@@ -44,10 +44,4 @@ $ kubectl delete -f {{< github_file >}}/samples/bookinfo/platform/kube/bookinfo-
 $ kubectl delete -f {{< github_file >}}/samples/sleep/sleep.yaml
 {{< /text >}}
 
-## Remove the Kubernetes Gateway API CRDs
-
-If you installed the Gateway API CRDs, remove them:
-
-{{< text bash >}}
-$ kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd/experimental?ref={{< k8s_gateway_api_version >}}" | kubectl delete -f -
-{{< /text >}}
+{{< boilerplate gateway-api-remove-crds >}}
```

Comment:
Don't you still want the heading?

Reply:
I'm gonna move this to the snippet, actually - ty for flagging.

Reply:
derp - the _header_ - yeah.